src/plugins/kbn_vislib_vis_types/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = function (kibana) {
 
   return new kibana.Plugin({
 
-    uiExports: {
-      visTypes: [
-        'plugins/kbn_vislib_vis_types/kbn_vislib_vis_types'
-      ]
+    ui: {
+      visTypes: './kbn_vislib_vis_types'
```

Comment:
I'm not opposed, we don't have a similar standard in the rest of the code though